### PR TITLE
Poc 367

### PR DESCRIPTION
**Pull request by drizzentic**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '2'
 services:
   etl-prod:
     container_name: etl-rest-server-prod
-    image: ampathke/etl-services:v2.16.5.2
+    image: ampathke/etl-services:POC-399
```
Comment (corneliouzbett) [inline docker-compose.yml:5]: Does this mean, we will be changing this in every PR??

File: programs/scope-builder.service.js [hunk 2]
```diff
@@ -291,10 +292,12 @@ function isInitialVisit(
   });
   // get latest initial encounter and compare with dateenrolled to check whether it's a new enrollment
   let latestEnc = initialEncounters[initialEncounters.length - 1];
+  console.log('has initial', initialEncounters, dateEnrolled);
```
Comment (corneliouzbett) [inline programs/scope-builder.service.js:295]: Please clean up the console logs
```suggestion
  let latestEnc = initialEncounters[initialEncounters.length - 1];
```

File: programs/scope-builder.service.js [hunk 3]
```diff
@@ -291,10 +292,12 @@ function isInitialVisit(
...
   if (
     initialEncounters.length > 0 &&
     dateEnrolled > latestEnc.encounterDatetime
   ) {
+    console.log('wapo');
```
Comment (corneliouzbett) [inline programs/scope-builder.service.js:300]: Ditto
```suggestion
```

**Review by corneliouzbett — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (drizzentic) [reply docker-compose.yml:5]: Ideally, it's supposed to be the docker image name for the deployed commit

**Review by corneliouzbett — APPROVED** [review]

**Review by corneliouzbett — APPROVED** [review]

**Pull request merged**